### PR TITLE
Section 6 Form: non-complex questions

### PR DESCRIPTION
**Pull request by julia-rom**
# Description

- Section 6 form excluding 6.2a/b/c/d (tabular input + complex questions)
- Added 6.1, 6.2, 6.3, 6.3a, and 6.4 (multiselect or drop down select)
- Does API call and loads data after submitting form
- 6.3a only appears if 6.3 is 'Yes'

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Instructions

1. Go to Site Interventions Form
2.  Click through sections
3. Submit
4. Reload to make sure data is saved

### CODE REVIEW

File: mrtt-ui/src/components/SiteInterventions.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react'
+import axios from 'axios'
+import { useParams } from 'react-router-dom'
+import {
+  useForm,
+  // useFieldArray,
```
Comment (mmnoo) [inline mrtt-ui/src/components/SiteInterventions.js:6]: guessing you will use this commented out code in the next step or so? If not, can you remove it and others?

File: mrtt-ui/src/components/SiteInterventions.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        toast.error(language.error.submit)
+      })
+  }
+  return isLoading ? (
+    <LoadingIndicator />
+  ) : (
+    <ContentWrapper>
+      <FormPageHeader>
+        <SectionFormTitle>Site Interventions</SectionFormTitle>
+        <Link to={-1}>&larr; {language.form.navigateBackToSiteOverview}</Link>
```
Comment (mmnoo) [inline mrtt-ui/src/components/SiteInterventions.js:96]: Im starting to wonder in the -1 we've been putting is the best idea. For instance with the new form nav, I could have gotten here from another form section. Maybe put in the path to the site overview.

Im getting rid of my -1 navs when I bump into them

**Review by mmnoo — COMMENTED** [review]
Approved with suggestion about navigating back.

File: mrtt-ui/src/components/SiteInterventions.js (hunk 2 above)
Comment (julia-rom) [reply mrtt-ui/src/components/SiteInterventions.js:96]: I think hiving the path to site overview is a better idea

**Comment by julia-rom** [comment]
I updated the site overview link + used the uncommented code in the next branch. Merging for now since I already branched off!

**Pull request merged**